pkg/tfcli/templates/main.tf.json.tpl

```diff
@@ -20,11 +20,6 @@
         "{{ .Resource.LabelType }}": {
             "{{ .Resource.LabelName }}": {
                 {{ .Resource.Body | printf "%s" }}
-                {{ if .Lifecycle.PreventDestroy -}}
```

Comment:
Are these relevant to this PR? Or just a refactor?

Reply:
Here we actually do some string manipulation on the printed form of the JSON, see the `,` before `prevent_destroy`. We'd have to unmarshal/marshal from `[]byte` to avoid doing that but now that we get to work with `map[string]interface{}`, we don't need any of this anymore. The additional property added directly to the map which is printed as is.